teuthology/packaging.py

```diff
@@ -455,6 +455,7 @@ def __init__(self, project, job_config, ctx=None, remote=None):
         # of roles instead, ctx is only used in _get_config_value_for_remote.
         self.ctx = ctx
         self.remote = remote
+        self.version_major = self.job_config.get('version_major', False)
```

Comment:
how about `install_per_major_version`? IMHO `version_major` is a little bit confusing.

Reply:
Yeah, looks better. Will fix it.